src/lib/Tab/LocationView/PagerLocationToDataMapper.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Tab\LocationView;
```

Comment:
I don't really see the relation between this class and Tab. I understand that it's used by LocationView tab, but it seems to be decoupled (good job!). The need to `exclude` in the DIC config should hint you that the placement is incorrect.